src/client/components/common/Charts/LineChart/index.tsx

```diff
@@ -117,7 +118,15 @@ export const LineChart: FC<LineChartProps> = ({ chartData, tooltipLabel, classNa
           return (
             <StyledTooltip align={isFirstHalf ? 'left' : 'right'}>
               <Text>{tooltipLabel || point.serieId}</Text>
-              <Text>{formatUsd(point.data.yFormatted.toString())}</Text>
+              <Text>
```

Comment:
I think we should create a const in this cases to keep the html clean.

`const customSymbolText = customSymbol ? formatAmount.... : formatUsd...;`

and then just do `<Text>{customSymbolText}</Text>